### ISSUE TITLE
A note of caution on model sims...

### ISSUE THREAD

**Issue by mbaker21231** [hunterecon2/epidemics#2]
In some of the models, one wishes to know how things ultimately turn out. For example, in the model with death rates, what one cares about is the total number of deaths. In these cases, one needs to ensure that there are enough time periods for things to have "run its course" or comparisons may be dubious!

**Comment by jhconning** [comment]
True, but presumably not so long that we might have realistically all dropped dead anyway!

**Comment by jhconning** [comment]
A  minor footnote to frame the death numbers.  NYC 5 boro population in 2019 approx 8.4 million

As of 4/14 this site https://projects.thecity.nyc/2020_03_covid-19-tracker/

says there have been 10367 "probable" covid19 deaths (higher than official stats, but based on death certificates).  So one in 810 NYC residents or 0.123 percent of the population.
